src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -1232,21 +1232,20 @@ void M4(int* p)
     }
 }";
             CreateCompilation(source, options: TestOptions.DebugExe.WithAllowUnsafe(true), parseOptions: TestOptions.Regular7_3).VerifyDiagnostics(
-                // (6,29): error CS8521: Pattern-matching is not permitted for pointer types.
+                // (6,29): error CS8524: Pointers cannot be matched to null in C# 7.3. Please use language version '8.0' or greater.
```

Comment:
This message doesn't look like what I would expect to be generated for this ErrorCode. Do the expected diagnostics need to be regenerated?

Reply:
Ping @kevinsun-dev

Reply:
Those... yeah. Those comments need redoing. One moment.